src/main/scala/io/iohk/ethereum/blockchain/sync/SyncUtils.scala

```diff
@@ -0,0 +1,11 @@
+package io.iohk.ethereum.blockchain.sync
+
+import io.iohk.ethereum.domain.BlockHeader
+
+trait SyncUtils {
```

Comment:
maybe we can move this `checkHeaders()` to `FastSyncBlocksValidationUtils` since header is part of a block, also could we change name of `checkHeaders()` to sth. like `checkHeadersChain` (or something more descriptive)?